distribution/test/itests/test-itests-dependencies-app/src/main/filtered-resources/features.xml

```diff
@@ -22,6 +22,11 @@
     <feature name="third-party-dependencies">
         <bundle>mvn:commons-io/commons-io/2.5</bundle>
         <bundle>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.hamcrest/${hamcrest-all.version}</bundle>
+
+        <bundle>wrap:mvn:org.codice.imaging.nitf/codice-imaging-nitf-core/${nitf-imaging.version}</bundle>
```

Comment:
If I don't I get: Caused by: org.osgi.framework.BundleException: Unable to build resource for mvn:org.codice.imaging.nitf/codice-imaging-nitf-render/0.7-SNAPSHOT: Unsupported 'Bundle-ManifestVersion' value: 1